src/clients/SnapshotGraphql.ts

```diff
@@ -36,38 +30,13 @@ export type SnapshotSpace = {
   strategies: SnapshotStrategy[]
 }
 
-export type SnapshotMessage<T extends string, P> = {
-  version: string
-  timestamp: string
-  space: string
-  type: T
-  payload: P
-}
-
-export type SnapshotRemoveProposalMessage = SnapshotMessage<'delete-proposal', { proposal: string }>
-export type SnapshotNewProposalPayload = {
-  name: string
-  body: string
-  end: Pick<Date, 'getTime'>
-  start: Pick<Date, 'getTime'>
-  snapshot: number
-  choices: string[]
-}
-
-export type SnapshotVotePayload = {
-  choice: number
-  metadata: unknown
-  proposal: string
-}
-
-export type SnapshotVoteMessage = SnapshotMessage<'vote', SnapshotVotePayload>
 export type SnapshotVoteResponse = SnapshotQueryResponse<{ votes: SnapshotVote[] }>
 export type SnapshotVote = {
-  id: string
+  id?: string
```

Comment:
why a Snapshot vote may not have an id?

Reply:
IDK, but they didn't have an id when I tried the request

```json
    {
        "voter": "0x3276bb5F7225b0a5244dbBeD70e21D9dcAaA5350",
        "created": 1658425499,
        "choice": 1,
        "proposal": {
          "id": "bafkreibk3amhbxkygift657vaxt5elihlbrtaz7ksfjuokiktinm6d4wka"
        }
      }
    ```